src/components/transaction-empty.tsx

```diff
@@ -2,22 +2,20 @@ import React from 'react';
 
 import {StyleSheet, View} from 'react-native';
 
-import {NoTransactionsIcon, Text} from './ui';
-
-import {LIGHT_GRAPHIC_SECOND_3, LIGHT_TEXT_SECOND_1} from '../variables';
+import {Color} from '@app/colors';
+import {NoTransactionsIcon, Text} from '@app/components/ui';
+import {I18N} from '@app/i18n';
 
 export const TransactionEmpty = () => {
   return (
-    <View style={page.container}>
-      <NoTransactionsIcon color={LIGHT_GRAPHIC_SECOND_3} style={page.space} />
-      <Text t14 style={{color: LIGHT_TEXT_SECOND_1}}>
-        No transactions
-      </Text>
+    <View style={styles.container}>
+      <NoTransactionsIcon color={Color.graphicSecond3} style={styles.space} />
```

Comment:
`NoTransactionsIcon` not handle Color type inside, use `getColor()`

Reply:
done